concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-95	95	guetig	kind	3299	KIND	1.511	6.25	5.75	6.833	6.083	6	6.458	6.229	1.917	2.167	2.25	2.273	2.042	2.261	2.149	5.25	5.5	4.833	4	5.375	4.417	4.896	3.833	3.583	5.25	4.182	3.708	4.739	4.213	5.083	4.583	5.667	5.083	4.833	5.375	5.104	4.083	3.833	3.833	4.091	3.958	3.957	3.957	5.813	5.938	1.083
+Gruehn-2008-200-96	96	harmlos	harmless			7.908	4.417	4.417	4.667	4.333	4.417	4.5	4.458	1.917	1.667	3	2.167	1.792	2.583	2.188	3.75	3.333	3.417	2.75	3.542	3.083	3.313	3.333	3.583	4	2.273	3.458	3.174	3.319	4.75	4.083	4	3.083	4.417	3.542	3.979	3.5	3.75	3.333	3.833	3.625	3.583	3.604	4.125	4.5	1.167
+Gruehn-2008-200-97	97	haesslich	ugly	1548	UGLY	3.174	1.75	2	2.333	2.25	1.875	2.292	2.083	4.25	4.1	4.417	4.5	4.625	4.458	4.542	3.25	2.75	2.917	3	3	2.957	2.979	6.333	6.75	5.417	6.833	6.542	6.125	6.333	2.083	1.75	2.417	2.833	1.917	2.625	2.271	3.25	3.167	3.083	3.667	3.208	3.375	3.292	2	3.188	1.708
+Gruehn-2008-200-98	98	heiter	bright			9.268	6.083	5.917	6.167	6.083	6	6.125	6.063	2.667	2.833	3	3.182	2.75	3.087	2.915	5.167	4.417	4.167	4.25	4.792	4.208	4.5	5.583	5.583	5.917	5.333	5.583	5.625	5.604	6	5.583	5.417	5.167	5.792	5.292	5.542	2.917	2.75	2.833	2.667	2.833	2.75	2.792	6.125	5.875	2
+Gruehn-2008-200-99	99	hektisch	hectic			3.988	2.5	2.417	1.75	2.25	2.458	2	2.229	6.583	6.25	6.25	5.917	6.417	6.083	6.25	2.833	2.667	3.917	1.917	2.75	2.917	2.833	5.5	6	5.083	5	5.75	5.043	5.404	4	3.25	2.667	2.75	3.625	2.708	3.167	2	1.833	2	1.727	1.917	1.87	1.894	2.688	2.875	1.417
+Gruehn-2008-200-100	100	hemmungslos	unrestrained			2.659	3.667	4.167	1.5	1.833	3.917	1.667	2.792	3.083	3.167	5.417	5.333	3.125	5.375	4.25	3.167	3.167	2.75	2.167	3.167	2.458	2.813	3.833	5.417	5.909	4.75	4.625	5.304	4.957	3.333	4.25	1.333	1.417	3.792	1.375	2.583	2.167	1.833	2.083	2.364	2	2.217	2.106	3.875	3.75	1.333
+Gruehn-2008-200-101	101	hilflos	helpless			9.555	2.583	2.417	2.25	2.667	2.5	2.458	2.479	5.667	5.5	5.818	4.833	5.583	5.304	5.447	1.333	1.083	1.818	2	1.208	1.913	1.553	4.167	4.917	4.667	4.917	4.542	4.792	4.667	2.25	2	2.083	1.75	2.125	1.917	2.021	4.25	4	3.75	3.833	4.125	3.792	3.958	1.563	2.5	1.75
+Gruehn-2008-200-102	102	hoeflich	polite	3300	POLITE	7.069	5.667	5.75	6.417	5.917	5.708	6.167	5.938	3.583	3.083	2.917	2.75	3.333	2.833	3.083	4.917	5.083	4.583	4.917	5	4.75	4.875	4.182	5.083	5.417	4.333	4.652	4.875	4.766	6.083	5.75	5.917	5.75	5.917	5.833	5.875	3.25	3.667	3.75	3.583	3.458	3.667	3.563	5.688	5.75	1
+Gruehn-2008-200-103	103	impulsiv	impulsive			543	5.417	5.083	5.25	5.182	5.25	5.217	5.234	4.75	5.25	4.917	5.667	5	5.292	5.146	4.25	2.917	4.25	3.75	3.583	4	3.792	4.417	3.917	5.25	4.75	4.167	5	4.583	5.167	4	6.25	4	4.583	5.125	4.854	1.917	1.833	2	1.833	1.875	1.917	1.896	4.313	4.438	1.625
+Gruehn-2008-200-104	104	intelligent	intelligent	2530	SMART	4.418	6.583	6.25	6.667	6.417	6.417	6.542	6.479	2.667	3.5	3.667	4.167	3.083	3.917	3.5	6.167	6.083	4.75	5.167	6.125	4.958	5.542	4.167	4.583	4.75	2.917	4.375	3.833	4.104	5.917	5.667	5.167	5.75	5.792	5.458	5.625	3	2.417	3	2.75	2.708	2.875	2.792	6.625	6.438	1.917
```

Comment:
I agree with the mapping. We've got two other instances of intelligent not being mapped to `2530 SMART`.

Reply:
Both instances belong to the same list (Hill-2015-999). The list also has an entry "smart" which is already mapped to `2530 SMART`. What's the best solution for me to try to do in this case?

Reply:
> Both instances belong to the same list (Hill-2015-999). The list also has an entry "smart" which is already mapped to `2530 SMART`. What's the best solution for me to try to do in this case?

Good point. Let's take this into a separate issue/PR.

Reply:
Ah, probably no need for a separte PR. That's the SimLex list. Yeah, double mappings are fine here due to the nature of the list I'd say.